storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -84,28 +86,28 @@ bool Redo_Log_Reader::find_start_checkpoint_lsn() {
 
     log_files_header_read(*log_sys, max_cp_field);
 
-    if (checkpoint_no_start !=
-        mach_read_from_8(log_sys->checkpoint_buf + LOG_CHECKPOINT_NO)) {
+    if (checkpoint_lsn_start !=
+        mach_read_from_8(log_sys->checkpoint_buf + LOG_CHECKPOINT_LSN)) {
       checkpoint_lsn_start =
           mach_read_from_8(log_sys->checkpoint_buf + LOG_CHECKPOINT_LSN);
-      checkpoint_no_start =
-          mach_read_from_8(log_sys->checkpoint_buf + LOG_CHECKPOINT_NO);
       continue;
     }
 
     break;
   }
 
+  */
```

Comment:
Remove this entire block, it is commented out

Reply:
not present in the latest commit.